src/lib/bootloader/grub2base.rb

```diff
@@ -435,14 +435,6 @@ def update_nvram_summary
 
       "#{_("Update NVRAM:")} #{status_string(update_nvram)} #{link}"
     end
-
-    def status_string(status)
```

Comment:
ugh, why it is removed? I see it is used upper.

Reply:
I have moved it to bootloader_base.rb because it will be used by different bootloaders now.